### PR TITLE
Workaround failing `semgrep` GitHub Action

### PR DESCRIPTION
**Pull request by ewbankkit**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Closes https://github.com/hashicorp/terraform-provider-aws/issues/25489.

### CODE REVIEW

**Comment by ewbankkit** [comment]
Can also [run `semgrep` directly](https://github.com/marketplace/actions/semgrep-action#update-from-april-11th-2022).

File: .github/workflows/semgrep-ci.yml [hunk 1]
```diff
@@ -14,13 +14,15 @@ on:
 
 jobs:
   semgrep:
+    name: Semgrep Code Quality Scan
     runs-on: ubuntu-latest
+    container:
+      image: returntocorp/semgrep-action
```
Comment (ankush) [inline .github/workflows/semgrep-ci.yml:20]: Does this actually work? The PR you raised is not from a fork, the action will use latest semgrep anyway AFAIK. 👀

Comment (ewbankkit) [reply .github/workflows/semgrep-ci.yml:20]: @ankush Thanks for spotting that. Image should be `returntocorp/semgrep`, not `returntocorp/semgrep-action`.

Comment (ewbankkit) [reply .github/workflows/semgrep-ci.yml:20]: https://github.com/hashicorp/terraform-provider-aws/pull/25508

**Pull request merged**